mqtt/mqtt-bridge/src/persist/loader.rs

```diff
@@ -45,12 +45,11 @@ where
         Self(inner)
     }
 
-    fn next_batch(&mut self) -> Result<VecDeque<(Key, Publication)>, PersistError> {
+    fn next_batch(&mut self) -> Result<VecDeque<(Key, Publication)>, StorageError> {
```

Comment:
just curious why you're replacing one error type with another?

Reply:
It doesn't really matter which one we choose, Persist and Storage were always going to merge based on what I was coding/aiming for. I just picked the one I was already used to typing for things. All the StorageErrors can become PersistErrors, just be ready for the explosion in ring buffer code  :)